tests/optimize/test_backtest_detail.py

```diff
@@ -251,6 +251,21 @@
             BTrade(sell_reason=SellType.STOP_LOSS, open_tick=2, close_tick=2)]
 )
 
+# Test 16: Buy, hold for 65 mins, then forcesell using roi=-1
+# Causes negative profit even though sell-reason is ROI.
+# stop-loss: 10%, ROI: 10% (should not apply), -100% after 65 minutes (limits trade duration)
+tc16 = BTContainer(data=[
+    # D  O     H     L     C     V    B  S
+    [0, 5000, 5025, 4975, 4987, 6172, 1, 0],
+    [1, 5000, 5025, 4975, 4987, 6172, 0, 0],
+    [2, 4987, 5300, 4950, 5050, 6172, 0, 0],
+    [3, 4975, 5000, 4940, 4962, 6172, 0, 0],  # ForceSell on ROI (roi=-1)
```

Comment:
why does it sell on the candle No.3, if tests_ticker_interval is '1h'?

Reply:
it buys on candle 1 (buy-signals are forwardshifted by one to have it aligned with real / dry mode).

because of this, candle 2 finishes after 60 minutes - and it sells 5 minutes into candle 3 (after 65 minutes).

Reply:
ah yeah, right, it buys at candle 1...